types/culori/src/wcag.d.ts

```diff
@@ -1,5 +1,7 @@
-import { Color } from "./common";
+import { Color } from "./common.js";
 
-export function luminance(color: Color | string): number;
+declare function wcagLuminance(color: Color | string): number;
```

Comment:
This might break code that, for some reason, depends directly on this file. Let’s hope no one is doing that.

Reply:
In previous versions that was probably possible! But now that culori ships `exports` in package.json, that only exposes a few imports, it’s impossible to import just this file directly. But yes if it exposed all files then you’d be correct.